src/plugins/otopi/dialog/human.py

```diff
@@ -139,7 +140,16 @@ def printline(line, newline=True):
             text = '\n'
         text = common.toStr(text)
 
-        lines = text.splitlines()
+        width = self._output_terminal_width() - 16
+
+        lines = [
+            l
```

Comment:
please use differnt variable name here, `l` is considered too similar to `i` and both of them are discouraged as variable names by pep

Reply:
Done.